### PR TITLE
📆 Fill in details about the panel discussion

### PR DESCRIPTION
**Pull request by drewbrew**
## Description

This adds a rough outline of the panel discussion as well as a blog post announcing the session.

Looking to announce this next week

### CODE REVIEW

File: _posts/2023-09-12-panel.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+-   Non-development individual contributor roles, such as design and tech writing
+-   Dev-adjacent roles, such as developer relations
+-   Going it alone: running your own consultancy shop
+-   Management roles, both technical and non-technical
+-   The advantages and challenges of specializing in Django in the current technical landscape
+
+Our panelists include (and more will be added before the session):
+
+-   Charles Rejonis, manager at Wharton
+-   [Elizabeth Christensen](/talks/postgres-performance-from-slow-to-pro/), manager at Crunchydata
```
Comment (elizabeth-christensen) [inline _posts/2023-09-12-panel.md:30]:
```suggestion
-   [Elizabeth Christensen](/talks/postgres-performance-from-slow-to-pro/), manager at Crunchy Data
```

Comment (drewbrew) [reply _posts/2023-09-12-panel.md]: thanks!

**Comment by tim-schilling** [comment]
@drewbrew is this defined enough for me to copy into Loudswarm's store of content? I can wait a few more days too.

**Comment by drewbrew** [comment]
@tim-schilling Yeah, go ahead. I'm going to merge and announce Monday.

**Comment by drewbrew** [comment]
@tim-schilling no, just the moderator

**Pull request merged**